include/game/bases/d_en_combo.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <game/bases/d_actor.hpp>
+
+class dEnCombo_c {
+public:
+    /// @unofficial
+    enum COMBO_TYPE_e {
+        COMBO_0,
+        COMBO_1,
+        COMBO_2
+    };
```

Comment:
Could better names be used here? We know 0 is no combo, 1 is the regular combo (7 claps) and 2 is the short combo (4 claps) for bigger enemies.